man/man8/zpool-split.8

```diff
@@ -30,9 +29,10 @@
 .Dd August 9, 2019
 .Dt ZPOOL-SPLIT 8
 .Os
+.
 .Sh NAME
 .Nm zpool-split
-.Nd Split devices off a ZFS storage pool creating a new pool
+.Nd split devices off ZFS storage pool creating a new pool
```

Comment:
nit: I'd keep the "a" here.

Reply:
As above, but went with "split devices off ZFS storage pool, creating new pool" for consistency (it definitely shows which pages I started on and which I did toward the end, it seems)